x/mint/keeper/keeper.go

```diff
@@ -0,0 +1,118 @@
+package keeper
+
+import (
+	"context"
+	"fmt"
+
+	"cosmossdk.io/core/store"
+	"cosmossdk.io/log"
+	"cosmossdk.io/math"
+
+	"github.com/babylonlabs-io/babylon/x/mint/types"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// Keeper of the mint store
+type Keeper struct {
+	cdc              codec.BinaryCodec
+	storeService     store.KVStoreService
+	stakingKeeper    types.StakingKeeper
+	bankKeeper       types.BankKeeper
+	feeCollectorName string
+	// the address capable of executing a MsgUpdateParams message. Typically, this
+	// should be the x/gov module account.
+	authority string
```

Comment:
so we are going to have params?

Reply:
This was from Celestia code base iirc. I guess they put it here for being future proof 😅